### ISSUE TITLE
Virus detected

### ISSUE THREAD

**Issue by danieloateslee** [ION28/BLUESPAWN#418]
Hi Support team,

Unfortunately, Microsoft Defender smart screen and other AV engines are detecting your .exe's as containing a TrojanDownloader:Win32/Cekar.gen!A.

Please check the virtual total web page

https://www.virustotal.com/gui/file/837b6d827746b9201ec8623008c9e69f3ece532c65484aee169f1ee9f5b8f245

**Comment by knowhatamine** [comment]
Yeah I'd really like to know how that assessment was made by these tools.

**Comment by ruppde** [comment]
> Yeah I'd really like to know how that assessment was made by these tools.

here you go: https://github.com/matterpreter/DefenderCheck

Takes a binary as input and splits it until it pinpoints that exact byte that Microsoft Defender will flag on, and then prints those offending bytes to the screen. This can be helpful when trying to identify the specific bad pieces of code in your tool/payload.
